### PR TITLE
Adding health-records to backend services and user serializer

### PR DESCRIPTION
**Pull request by saneshark**
Not entirely sure at what point this should get merged, seeing as we're not ready to have these services available in production, but here it is as a PR when we're ready to merge.

### CODE REVIEW

**Comment by saneshark** [comment]
Also, are we comfortable calling it health-records instead of blue-button?

**Comment by saneshark** [comment]
Fixes https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1100

File: app/serializers/user_serializer.rb [hunk 1]
```diff
@@ -30,11 +30,17 @@ def services
...
     service_list << BackendServices::EVSS_CLAIMS if object.can_access_evss?
     service_list << BackendServices::USER_PROFILE if object.can_access_user_profile?
     service_list
   end
+
+  private
+
+  def mhv_based_services
+    [BackendServices::RX, BackendServices::MESSAGING, BackendServices::HEALTH_RECORDS]
+  end
```
Comment (aub) [inline app/serializers/user_serializer.rb]: Why not have this be a constant instead of a method?

**Comment by saneshark** [comment]
@aub I took your comment to mean 'add the constant to' backend_services.rb as opposed to a constant in user_serializer.rb

**Pull request merged**